src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -15,6 +15,10 @@
  <div id="container">
    <h1>Register</h1>
 
+   <% if(request.getAttribute("error") != null){ %>
+       <h2 style="color:red"><%= request.getAttribute("error") %></h2>
+   <% } %>
+
```

Comment:
Nit: I realize it's indented like this on the project page, but let's make our best effort to keep indentation consistent (e.g. 2 spaces for all nested elements).